packages/core/src/fields/types/relationship/views/RelationshipSelect.tsx

```diff
@@ -88,6 +88,21 @@ export function useFilter (search: string, list: ListMeta, searchFields: string[
       })
     }
 
+		for (const fieldKey of relationsSearchFields) {
+      const [relation, relationsSearchField] = fieldKey.split('.')
+      const field = list.fields[relation]
+
+      conditions.push({
+        [field.path]: {
+          some:{
```

Comment:
`some` is only for one to many relationship, I believe this does not work for 1:1 relationship.

Reply:
Indeed, will think of something